packages/mobile/src/transactions/send.ts

```diff
@@ -31,6 +31,7 @@ const ALWAYS_FAILING_ERROR = 'always failing transaction'
 const KNOWN_TX_ERROR = 'known transaction'
 
 const getLogger = (tag: string, txId: string) => {
+  let startTime = new Date().getTime()
```

Comment:
I think `Date.now()` is a shorter-hand version of this same thing. Avoids need to all the `new` keywords

Reply:
Woah, awesome!